src/khoj/interface/web/config.html

```diff
@@ -478,28 +475,16 @@ <h3 class="card-title">
         })
     }
 
-    var configure = document.getElementById("configure");
-    configure.addEventListener("click", function(event) {
-        event.preventDefault();
-        updateIndex(
-            force=false,
-            successText="Saved!",
-            errorText="Unable to configure. Raise issue on Khoj <a href='https://github.com/khoj-ai/khoj/issues'>Github</a> or <a href='https://discord.gg/BDgyabRM6e'>Discord</a>.",
-            button=configure,
-            loadingText="Saving...",
-            emoji="💾");
-    });
-
-    var reinitialize = document.getElementById("reinitialize");
-    reinitialize.addEventListener("click", function(event) {
+    var sync = document.getElementById("sync");
+    sync.addEventListener("click", function(event) {
         event.preventDefault();
         updateIndex(
             force=true,
-            successText="Reinitialized!",
-            errorText="Unable to reinitialize. Raise issue on Khoj <a href='https://github.com/khoj-ai/khoj/issues'>Github</a> or <a href='https://discord.gg/BDgyabRM6e'>Discord</a>.",
-            button=reinitialize,
-            loadingText="Reinitializing...",
-            emoji="🔄");
+            successText="Synced!",
+            errorText="Unable to sync. Raise issue on Khoj <a href='https://github.com/khoj-ai/khoj/issues'>Github</a> or <a href='https://discord.gg/BDgyabRM6e'>Discord</a>.",
+            button=sync,
+            loadingText="Syncing...",
+            emoji="♻");
```

Comment:
Per the [issue description](https://github.com/khoj-ai/khoj/issues/700), can you also add a better icon here? This one would work: https://www.svgrepo.com/svg/489800/sync. You can see some of the other assets in the file (such as `copy-solid.svg`) for reference of how to use this.